dietpi/dietpi-software

```diff
@@ -10464,30 +10464,31 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
 
 			# - run syncthing to create cert/config and exit
-			syncthing -generate="/$HOME/.config/syncthing"
+			syncthing -generate="$FP_DIETPI_USERDATA_DIRECTORY"/syncthing
 
 			# - Disable automatic upgrades
-			sed -i '/<\/autoUpgradeIntervalH>/c\        <autoUpgradeIntervalH>0<\/autoUpgradeIntervalH>' "$HOME"/.config/syncthing/config.xml
+			sed -i '/<\/autoUpgradeIntervalH>/c\        <autoUpgradeIntervalH>0<\/autoUpgradeIntervalH>' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# - Allow external access (LAN).
-			sed -i '/:8384<\/address>/c\        <address>0.0.0.0:8384<\/address>' "$HOME"/.config/syncthing/config.xml
+			sed -i '/:8384<\/address>/c\        <address>0.0.0.0:8384<\/address>' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# - Set default folder
-			sed -i '/label=\"Default Folder/c\    <folder id=\"0000-0000\" label=\"DietPi user data\" path=\"/mnt/dietpi_userdata/\" type=\"readwrite\" rescanIntervalS=\"60\" ignorePerms=\"false\" autoNormalize=\"true\">' "$HOME"/.config/syncthing/config.xml
+			sed -i '/label=\"Default Folder/c\    <folder id=\"0000-0000\" label=\"syncthing data\" path=\"/mnt/dietpi_userdata/syncthing_data\" type=\"readwrite\" rescanIntervalS=\"60\" ignorePerms=\"false\" autoNormalize=\"true\">' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# - Disable browser starting
-			sed -i '/<\/startBrowser>/c\        <startBrowser>false<\/startBrowser>' "$HOME"/.config/syncthing/config.xml
+			sed -i '/<\/startBrowser>/c\        <startBrowser>false<\/startBrowser>' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# services
 			cat << _EOF_ > /etc/systemd/system/syncthing.service
 [Unit]
 Description=Syncthing
 After=network.target
 
+
 [Service]
 Type=simple
-ExecStart=/usr/bin/syncthing -logfile=/var/log/syncthing.log -logflags=3
```

Comment:
@joaofl 

```
dietpi@DietPi:~$ echo 1 > /var/log/test.log
-bash: /var/log/test.log: Permission denied
```

Solution would be to add logfile create/permissions to systemD service, eg:
```
ExecStartPre=/bin/bash -c 'echo 1 > /var/log/syncthing-inotify.log'
ExecStartPre=/bin/bash -c 'chown dietpi:dietpi /var/log/syncthing-inotify.log'
ExecStart=/usr/bin/syncthing-inotify -logfile=/var/log/syncthing-inotify.log -logflags=3 -home=$FP_DIETPI_USERDATA_DIRECTORY/syncthing
```